packages/shared/src/utils/helpers/format-response.ts

```diff
@@ -42,6 +43,33 @@ export const formatPortfolioPosition = (
     };
 };
 
+export type TIDVErrorStatus = typeof idv_error_statuses[keyof typeof idv_error_statuses];
+
+//formatIDVError is parsing errors messages from BE (strings) and returns error codes for using it on FE
+export const formatIDVError = (errors: string[], status_code: string) => {
+    if (errors.length === 0 && status_code === 'none') return null;
+    const error_keys: Record<string, TIDVErrorStatus> = {
+        name: 'POI_NAME_MISMATCH',
+        birth: 'POI_DOB_MISMATCH',
+        rejected: 'POI_FAILED',
+    };
+    if (status_code === 'expired') {
+        return 'POI_EXPIRED';
+    }
+    const status: TIDVErrorStatus[] = [];
+    errors.forEach(error => {
+        const error_regex = RegExp(/(name|birth|rejected)/i).exec(error);
+        if (error_regex) {
+            status.push(error_keys[error_regex[0].toLowerCase()]);
+        }
+    });
+    return status.includes(error_keys.name) &&
+        status.includes(error_keys.birth) &&
+        !status.includes(error_keys.rejected)
+        ? 'POI_NAME_DOB_MISMATCH'
+        : status[0] ?? 'POI_FAILED';
```

Comment:
clarification: is this a Business logic?

Reply:
Yes. Currently it is on FE side. There is a card created for BE to provide this in response